src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java

```diff
@@ -56,7 +56,31 @@ public class HydrateBreakMessageDictionary {
                         add("Dehydration can cause dry mouth, lips, and eyes. Take a hydration break");
                     }}
             );
-
+    
+    /**
+     * Hydrate Reminder interval break text to display in caring form
+     */
+    private static final List<String> HYDRATE_BREAK_CARING_TEXT_LIST =
+            Collections.unmodifiableList(
+                    new ArrayList<String>() {{
+                        add("Drink water before it's too late.");
```

Comment:
The plugin automatically adds `, {username}.` at the end of each message. Can you remove all the ending punctuation for each message?

For example, this first message will be displayed as `Drink water before it's too late., jmakhack.`.

Reply:
Removed all the punctuation.